src/foam/nanos/ticket/Ticket.js

```diff
@@ -77,7 +78,7 @@ foam.CLASS({
     },
     {
       name: 'COMMENT_NOTIFICATION',
-      message: 'A ticket assigned to you has been updated'
+      message: 'A comment has been added to a ticket assigned to you: '
```

Comment:
We send this out on all updates, no? 
An attachment added, but no comment would trigger a notification, would it not? 
The original  wording works for all cases. 

Just add a period and then the ticketId. 
so  original message     message. ticketId

Reply:
To trigger a notification on attachments, I created a new rule on the nanopay side that is triggered when a new attachment is added to the ticket.

Reply:
Fine, but the old language works for all cases. and will continue to be correct when we fix the notification so that a notification is sent on attachment added.